src/main/resources/application.properties

```diff
@@ -341,3 +341,23 @@ snyk.thread.timeout.duration=60
 # This value is in seconds. The default value is 60 seconds.
 snyk.limit.refresh.period=60
 
+# Optional
+#Defines the maximum number of purl sent in a single request to OSS Index.
+# The default value is 128.
+ossindex.request.max.purl=128
+
+# Optional
+#Defines the maximum number of attempts used by Resilience4J for exponential backoff retry regarding OSSIndex calls.
+# The default value is 50.
+ossindex.retryer.backoff.max.attempts=50
+
+# Optional
+#Defines the multiplier used by Resilience4J for exponential backoff retry regarding OSSIndex calls.
+# The default value is 2.
+ossindex.retryer.backoff.multiplier=2
+
+# Optional
+#Defines the maximum duration used by Resilience4J for exponential backoff retry regarding OSSIndex calls. This value is in milliseconds
+# The default value is one hour.
```

Comment:
One hour seems a bit excessive, considering this will block a thread for the entire time. IMO it's OK to fail when the retry didn't yield any success after just a few minutes at max. WDYT?

Reply:
What happens if a call fails even after multiple retries? Will the analysis just be incomplete "forever"for these components, or will there be a retry anyway on the next run of the analyzer?

Reply:
> One hour seems a bit excessive, considering this will block a thread for the entire time. IMO it's OK to fail when the retry didn't yield any success after just a few minutes at max. WDYT?

I agree. That was quite excessive. I'll set it to ten minutes.

Reply:
> What happens if a call fails even after multiple retries? Will the analysis just be incomplete "forever"for these components, or will there be a retry anyway on the next run of the analyzer?

There will be a retry at the next run of the global portfolio analysis which happens by default every 24H.